docker-compose.yml

```diff
@@ -6,8 +6,9 @@ networks:
       config:
         - subnet: 172.33.0.0/16
 services:
-    wifi.public.dappnode.eth:
+    wifi.dnp.dappnode.eth:
         image: 'wifi.dnp.dappnode.eth:0.0.1'
+        container_name: 'DAppNodeCore-wifi.dnp.dappnode.eth'
         build: ./build
         volumes:
```

Comment:
If we continue with the idea of making it a core package and it uses environment variables we must add:
```
    env_file:
      - wifi.dnp.dappnode.eth.env
```
and create this file with this content:
```
SSID=DAppNodeWIFI
WPA_PASSPHRASE=dappnode
NAT=true
INTERFACE=
CHANNEL=11
```